src/App/ViewModels/JobDetailViewModel.cs

```diff
@@ -15,18 +15,18 @@ public class JobDetailViewModel : INotifyPropertyChanged
         public JobDetailModel JobsDetail { get; set; }
         public Command OpenJobWebURLCommand { get; set; }
 
-        public JobDetailViewModel()
+        public JobDetailViewModel(string Link)
         {
 
-            new Action(async () => await LoadData())();
+            new Action(async () => await LoadData(Link))();
 
             OpenJobWebURLCommand = new Command<string>(OpenJobWebURL);
         }
-        public async Task LoadData()
+        public async Task LoadData(string link)
```

Comment:
The name of the method should end in Async -> LoadDataAsync

Reply:
@sact1909 ^

Reply:
Listo, tambien le agregue lo mismo a otros metodos que encontre que tenian el mismo caso...